OmniKit/PumpManager/PodState.swift

```diff
@@ -21,6 +21,10 @@ public enum SetupProgress: Int {
     case completed
     case activationTimeout
     case podIncompatible
+
+    public var primingNeverAttempted: Bool {
```

Comment:
Yes. This is as intended, as the name indicates, and is indeed the desired behavior. Once a prime is attempted (independent of whether or not the pod actually hears it), Loop should never again send a FaultConfig command to this pod since the FaultConfig has already been successfully completed and doing it again risks an 049 pod fault.